### ISSUE TITLE
BigQuery Sharded Tables

### ISSUE THREAD

**Issue by Sgiostra-BitBang** [dataform-co/dataform#1442]
There is a feature in Dataform that allow to use [**Sharded Tables**](https://cloud.google.com/bigquery/docs/partitioned-tables?hl=en#dt_partition_shard) as source?

**Comment by bigrogerio** [comment]
Great question. But why not partitioning instead sharding? I think sharding is not the best practice in BigQuery nowadays, IMO.

I have worked with sharded tables with Analytics 360. but i think that it's outdated now. 

With partitioning, you can have up to 4000 partitions, whereas when you query date sharded tables, you can only query up to 1000 tables at once.

Best regards.

**Comment by astorrs** [comment]
Yes you can do this: https://github.com/dataform-co/dataform/discussions/1496
